lib/srfi/60/60.scm

```diff
@@ -405,9 +405,9 @@
 ;;---------------------------------------------------------------------------
 ;; reverse-bit-field
 
-(define-prim (##reverse-bit-field n start end)
+(define-prim (reverse-bit-field n start end)
 
-  (define (bit-reverse k n)
+  (define (fixnum-bit-reverse k n)
     (let reverse-loop ((m (if (##fxnegative? n)
```

Comment:
I've looked at Aubrey's code, and I don't understand why he pre-treats the argument n and then post-processes the result when n is negative.  Wouldn't he get the same result without this pre- and post-treatment?